examples/memgpt_client.py

```diff
@@ -0,0 +1,47 @@
+from memgpt import create_client, Admin
+from memgpt.constants import DEFAULT_PRESET, DEFAULT_HUMAN, DEFAULT_PERSONA
+
+
+"""
+Make sure you run the MemGPT server before running this example.
+```
+export MEMGPT_SERVER_PASS=your_token
+memgpt server
+```
+"""
+
+
+def main():
+    # Create an admin client
+    admin = Admin(base_url="http://localhost:8283", token="your_token")
```

Comment:
We could alternatively call "token" here "password" instead, to avoid confusion with the user-level API key "tokens". Can be done in another PR (if desired) though.